charts/db-operator/ci/ci-2-values.yaml

```diff
@@ -0,0 +1,124 @@
+image:
+  repository: ghcr.io/kloeckner-i/db-operator
+  pullPolicy: Always
+
+reconcileInterval: "60"
+
+rbac: {}
+
+serviceAccount: {}
+
+resources: {}
+
+nodeSelector: {}
+
+annotations: {}
+
+affinity:
+  nodeAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 1
+        preference:
+          matchExpressions:
+            - key: another-node-label-key
+              operator: In
+              values:
+                - another-node-label-value
+tolerations:
+  - key: "key1"
+    operator: "Equal"
+    value: "value1"
+    effect: "NoSchedule"
+  - key: "key2"
+    operator: "Equal"
+    value: "value2"
+    effect: "NoSchedule"
+
+config:
+  instance:
+    google:
+      proxy:
+        nodeSelector: {}
+        image: kloeckneri/db-auth-gateway:0.1.7
+        metricsPort: 9090
+    generic: {}
+    percona:
+      proxy:
+        image: severalnines/proxysql:2.0
+        metricsPort: 9090
+  backup:
+    activeDeadlineSeconds: 600  # 10m
+    nodeSelector:
+      nodetype: database
+    postgres:
+      image: "kloeckneri/pgdump-gcs:latest"
+    mysql:
+      image: "kloeckneri/mydump-gcs:latest"
+  monitoring:
+    promPushGateway: ""
```

Comment:
always empty. would not test this setting.

Reply:
Added this value to ci-2-values and ci-3-values
```
promPushGateway: prometheus-pushgateway.monitoring:9090
```